api/src/main/java/org/apache/iceberg/LockManager.java

```diff
@@ -17,14 +17,14 @@
  * under the License.
  */
 
-package org.apache.iceberg.aws.glue;
+package org.apache.iceberg;
```

Comment:
@jackye1995, couldn't this still be in the `util` package even if it is in the API module?

Reply:
yes it can still be in the `util` package. 

I think the actual module and package of this class depends on how we want to position this feature. I remember you mentioned we might want to eventually introduce some kind of pessimistic lock to fully lock a table to make certain operations succeed.

So the following makes sense to me:
1. if it's only for `HadoopTables` (or maybe `HadoopCatalog`), then `iceberg-core` module can work and it can either be in `org.apache.iceberg.hadoop` or `org.apache.iceberg.util`.
2. if we want this to do other things in the future like fully locking a table in a catalog, then I think `iceberg-api` module and `org.apache.iceberg` package seems more official.

Reply:
I think that having this in the API module makes sense, but I would put `LockManagers` and the in-memory implementation in core. I'd also move the implementation back to the `util` package since we want to avoid making `org.apache.iceberg` any more huge.